service_mesh/v2x/ossm-kiali-ossmc-plugin.adoc

```diff
@@ -0,0 +1,29 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="ossm-kiali-ossmc-plugin"]
+= OpenShift Service Mesh Console plugin
+include::_attributes/common-attributes.adoc[]
+:context: ossm-kiali-ossmc
+
+toc::[]
+
+The {SMProductName} (OSSMC) plugin is an extension to the {product-title} web console that provides visibility into your Service Mesh. With the OSSMC plugin installed, a new *Service Mesh* menu option is available in the left-hand navigation of the web console, as well as new *Service Mesh* tabs that enhance the existing *Workloads* and *Services* console pages.
+
+[IMPORTANT]
+====
+If you are using a certificate that your browser does not initially trust, you must tell your browser to trust the certificate first before you are able to access the OSSMC plugin. To do this, go to the Kiali standalone user interface (UI) and tell the browser to accept its certificate.
+====
+
+include::modules/ossm-kiali-ossmc-plugin-user-guide.adoc[leveloffset=+1]
+
+include::modules/ossm-kiali-ossmc-plugin-install-web-console.adoc[leveloffset=+1]
+
+include::modules/ossm-kiali-ossmc-plugin-install-cli.adoc[leveloffset=+1]
+
+include::modules/ossm-kiali-ossmc-plugin-uninstall-web-console.adoc[leveloffset=+1]
+
+include::modules/ossm-kiali-ossmc-plugin-uninstall-cli.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+[id="additional-resources_ossm-kiali-ossmc-plugin"]
+== Additional resources
+* To explicitly declare the Kiali instance you want, see: link:<https://kiali.io/docs/configuration/ossmconsoles.kiali.io/#.spec.kiali.serviceNamespace.>[OSSMConsole CR Reference]
```

Comment:
```suggestion
* link:https://kiali.io/docs/configuration/ossmconsoles.kiali.io/#.spec.kiali.serviceNamespace[.spec.kiali.serviceNamespace]
```

See https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#additional-resources-sections re "no text outside of the square brackets" and how to format links